app/core_behaviour.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Make Atom the default feed format, and remove other formats from <head>
+ */
+
+add_action('init', function() {
+    add_filter('default_feed', function() { return 'atom'; });
+
+    remove_action('do_feed_rdf', 'do_feed_rdf', 10, 1);
+    remove_action('do_feed_rss', 'do_feed_rss', 10, 1);
+    remove_action('do_feed_rss2', 'do_feed_rss2', 10, 1);
+    // Remove Emoji script
+    remove_action('wp_head', 'print_emoji_detection_script', 7);
+    remove_action('wp_print_styles', 'print_emoji_styles');
+    remove_action('admin_print_styles', 'print_emoji_styles');
+    remove_action('admin_print_scripts', 'print_emoji_detection_script');
+    // Remove extra crap from wp_head
+    remove_action('wp_head', 'rsd_link');
+    remove_action('wp_head', 'wp_generator');
+    remove_action('wp_head', 'wlwmanifest_link');
+    remove_action('wp_head', 'feed_links_extra', 3);
+    remove_action('wp_head', 'start_post_rel_link', 10, 0);
+    remove_action('wp_head', 'parent_post_rel_link', 10, 0);
+    remove_action('wp_head', 'adjacent_posts_rel_link', 10, 0);
+});
+
+add_action('wp_head', function () {
+    ?>
+    <link rel="alternate" type="application/atom+xml" title="<?php echo get_bloginfo('name') ?> Feed" href="<?php echo esc_attr(get_feed_link('atom')) ?>">
+    <?php
+});
```

Comment:
This file has been renamed to `app/CoreBehaviour.php`. Sorry, you'll need to take the extra bits you've added and add them there.